cirq/experiments/qubit_characterizations.py

```diff
@@ -0,0 +1,544 @@
+import itertools
+from typing import Sequence, Tuple
+from cirq import circuits, devices, ops, protocols, sim, study, value
+import numpy
+from matplotlib import pyplot
+from mpl_toolkits.mplot3d import Axes3D
+
+
+class RabiResult(object):
+    """Class for storing and plotting data from a Rabi oscillation
```

Comment:
`Class for` is redundant. Just `Results from a Rabi oscillation experiment.` is fine.

Reply:
Done

Reply:
Done